src/frontends/tensorflow/src/frontend.cpp

```diff
@@ -62,6 +62,9 @@ bool FrontEnd::supported_impl(const std::vector<ov::Any>& variants) const {
         std::string model_path = variants[0].as<std::string>();
         if (ov::util::ends_with(model_path, suffix.c_str())) {
             return true;
+        } else if (ov::util::directory_exists(model_path) && ov::util::file_exists(model_path + "/saved_model.pb") &&
+                   ov::util::file_exists(model_path + "/variables/variables.index")) {
```

Comment:
model_path + "/variables/variables.index" is not necessary to exist. I remember SM without these files. Please double-check

Reply:
Yes, looks like you are right, it might be some kind of "program-only" file. Applied.